release-notes/docs/3.8.0.md

```diff
@@ -0,0 +1,180 @@
+# 3.8.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Tasks data available for analysis](#tasks-data-available-for-analysis)
+- [Create Places by SMS](#create-places-by-sms)
+- [Replication improvements](#replication-improvements)
+- [Performance](#performance)
+- [And more...](#and-more)
+
+## Known Issues
+
+No known issues.
+
+## Upgrade Notes
+
+### Breaking changes
+
+The changes to [tasks and targets](#task-and-target-data-available-for-analysis) require a "3.8 compatible" partner configurations to be deployed for the tasks and target tabs to continue to function. You'll see errors on these tabs and the console error `Rules Engine: Updates to the nools schema are required` if the configuration is not cmpatible with 3.8. All "3.8 compatible" configurations are backward compatible and can be deployed safely to any Core Framework version. It is therefore recommended to deploy the "3.8 compatible" configuration before upgrading so your application will work seamlessly once the upgrade completes.
+  * For partners using declarative configuration, ensure your configuration is deployed using [medic-conf 3.1.0](https://github.com/medic/medic-conf/blob/master/release-notes.md#31) or greater.
+  * For partners maintaining their configuration in the legacy `rules.nools.js` file, ensure the latest [schema changes](https://github.com/medic/medic-projects/commit/3bb4cd5ade155cb7eca0e0ea6791cc82796d3949) are deployed.
+
+These changes also result in new constraints for partner application task and target code:
+
+* Tasks with multiple `actions` cannot use different contacts for their `actions[n].content.contact` values. For example, one action cannot open a form for contact `x` and a second action for the same task open a form for contact `y`.
+* Reports "with errors" are now available in all task and target calculations (instead of just some calculations) and partner code is responsible for filtering them as necessary.
+* For partners maintaining their configuration in a `rules.nools.js` file. Multiple contacts emitting tasks with conflicting `id` values is no longer supported. This will result in errant behavior in 3.8.0.
```

Comment:
```suggestion
* For partners maintaining their configuration in a `rules.nools.js` file, multiple contacts emitting tasks with conflicting `id` values is no longer supported. This will result in errant behavior in 3.8.0.
```